apps/yara/CMakeLists.txt

```diff
@@ -14,11 +14,6 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-if (COMPILER_MSVC)
-    message (STATUS "Yara is currently not build on Microsoft Visual because it requires OpenMP 3.0.")
-    return ()
-endif ()
-
```

Comment:
This is not true anymore?

Reply:
I don't know, I'll see what the CI says.

It skips MSVC because of OpenMP, but a few lines later it says there is no multi-threading if there is no OpenMP. So there is either no reason to skip MSVC or it's different from what's written here.

Reply:
Yara works fine with a small hack; otherwise there are some problems with ambiguous calls
Fiona compiles fine, but 2 out of 4 tests fail